source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -313,6 +314,9 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
                         telemetryProps["multilineLanguages"] = @clusterMultilineLanguages
                       end
                     end
+                    if (!clusterNetworkFlowLogsEnabled.nil? && !@clusterNetworkFlowLogsEnabled.empty?)
+                      telemetryProps["networkFlowLogsEnabled"] = @clusterNetworkFlowLogsEnabled
```

Comment:
We should also include throttle configuration as part of the telemetry.

Reply:
added